generated/nidcpower/nidcpower/lcr_load_compensation_spot.py

```diff
@@ -0,0 +1,118 @@
+import ctypes
+import numbers
+import platform
+
+import nidcpower._visatype
+import nidcpower.enums as enums
+
+
+# This class is an internal ctypes implementation detail that corresponds to
+# NILCRLoadCompensationSpot in the C API
+class struct_NILCRLoadCompensationSpot(ctypes.Structure):  # noqa N801
+    if platform.system() == "Windows" and platform.architecture()[0] == "64bit":
+        _pack_ = 8
+    elif platform.system() == "Linux" or platform.system() == "Darwin":
+        _pack_ = 4
+    else:
```

Comment:
As explained earlier in https://github.com/ni/nimi-python/pull/1702#discussion_r919946197, this is following the exact logic in the public "nidcpower.h" header file for NI-DCPower 21.8, which has already been released, so I don't think it can be fixed by a quick PR.